beacon-chain/core/blocks/block_operations.go

```diff
@@ -394,6 +412,9 @@ func ProcessProposerSlashings(
 ) (*stateTrie.BeaconState, error) {
 	var err error
 	for idx, slashing := range body.ProposerSlashings {
+		if slashing == nil {
+			continue
```

Comment:
@nisdas @terencechain i think it should fail. why should a block contain nil entries?

Reply:
changed to fail